source/change-streams/tests/change-streams-resume-whitelist.json

```diff
@@ -1648,6 +1648,103 @@
           }
         ]
       }
+    },
+    {
+      "description": "change stream resumes after CursorNotFound",
+      "minServerVersion": "4.2",
+      "maxServerVersion": "4.2.99",
```

Comment:
If I remove this line limiting the maxServerVersion, the test still passes using MongoDB 4.4.  Can you explain why we need the other 4.4-specific test?

Reply:
Good point. Tests for the other errors are duplicated between the two files because we need to use a different failpoint on 4.4+ to get the error labels added, but that's not necessary for this specific error. I removed the 4.4-specific test and took out the `maxServerVersion` line for this test in my next commit.